recipe/meta.yaml

```diff
@@ -65,18 +73,9 @@ outputs:
         - hdf5
         - hdf5 * {{ mpi_prefix }}_*
       run:
-        - {{ compiler('cxx') }}  # [linux]
-        - cmake >=3.9
+        # only need library dependencies that _lack_ run_exports here
         - eigen
-        - {{ mpi }}
         - parmetis
-        - petsc
```

Comment:
Cleaned up lots of dependencies. The only things that need to be in run are those that _won't_ get added by run_exports. That's only eigen and parmetis now.